libtorch/release.nix

```diff
@@ -3,93 +3,55 @@
 with pkgs;
 
 let
-  callCpu = opts: callPackage ./generic.nix ({ mklSupport = true; } // opts);
+  libmklml = opts: callPackage ./mklml.nix ({
+  } // opts);
+  callCpu = opts: callPackage ./generic.nix ({
+    mklml = libmklml {};
+    libcxx = libcxx;
+  } // opts);
   callGpu = opts: callPackage ./generic.nix ({
-    mklSupport = true;
-    cudaSupport = true;
+    mklml = libmklml {};
+    libcxx = libcxx;
   } // opts);
 in
-
 {
+  libmklml = libmklml {};
   libtorch_cpu = callCpu {
-    version = "1.1";
+    version = "1.2";
+    buildtype = "cpu";
     mkSrc = buildtype:
       if stdenv.hostPlatform.system == "x86_64-linux" then
         fetchzip {
-          url = "https://download.pytorch.org/libtorch/${buildtype}/libtorch-shared-with-deps-latest.zip";
-          sha256 = "09iwdy31zg9dzkrjx8mwpds9mxrv775msn01v1njkpjymvi7llz6";
+          url = "https://download.pytorch.org/libtorch/cpu/libtorch-cxx11-abi-shared-with-deps-1.2.0.zip";
+          sha256 = "0vh5fw9h1rydp2bbrlkq54z29p1v0lpfmllddk82sgz7sr3jld66";
         }
       else if stdenv.hostPlatform.system == "x86_64-darwin" then
-        fetchzip {
-          url = "https://download.pytorch.org/libtorch/${buildtype}/libtorch-macos-1.1.0.zip";
-          sha256 = "03wqgvmyz2dv5iin27rnhxy6blk7gf0h49vgwmnab9c5j43y2y3d";
+        (import <nixpkgs> {}).fetchzip {
```

Comment:
I do not want to use <nixpkgs>, but if we do not use it, nix-build is failed. (unzip outputs "error: invalid zip file with overlapped components (possible zip bomb)")
https://github.com/junjihashimoto/hasktorch/commit/4f1adef834dfc26a6c04312eaa38335939f84fd4/checks

Reply:
The download does not work on linux, tool.
I may have to use <nixpkgs> on linux.
https://circleci.com/gh/junjihashimoto/hasktorch/85